proto/frequenz/api/microgrid/microgrid.proto

```diff
@@ -265,7 +269,7 @@ service Microgrid {
   //
   // If any of the above mentioned actions for a given component has already
   // been performed, then this method call efffectively skips that action.
-  rpc ColdStandby(ComponentIdParam) returns (google.protobuf.Empty) {
+  rpc ColdStandbyComponent(ComponentIdParam) returns (google.protobuf.Empty) {
```

Comment:
Not sure where ColdStandby comes from.
If the Cold refers to the Component, maybe ColdComponentStandby, otherwise disregard this comment

Reply:
Good question!
The verb here is `ColdStandby`. For inverters it means that AC relays get opened. Inverters take about a minute to become operation from this state. This differs from `HotStandby`, where we put inverters in a state where we can make them operation again in a few milliseconds.

Reply:
I'd vote for `{Hot, Cold}ComponentStandy`. It's the `ComponentStandby` RPC and we distinguish between the Hot and Cold variants.

Reply:
No sure about that. The verb here is `{Hot,Cold}Standby`, and we are following the verb_entity pattern. `{Hot,cold}ComponentStandby` breaks that pattern.

Reply:
I agree that cold-standby and hot-standby are too strong of a term to split. Another alternative would be not to use it as a verb and use something like `SetComponentColdStandby`.

Reply:
That would mean the other RPCs become `SetComponentStart`, `SetComponentStop`, etc.

Another alternative could be `SetComponentStateX`. But I would go for it only if you really really dislike `HotStandby` and `ColdStandby` :)

Note that `standby` is a legit verb to use here. Maybe we get rid of both the commands and just keep `Standby`? Then we have 3 clean verbs `Start`, `Standby`, and `Stop`? `HotStandby` get renamed to `Standby`, and instead of `ColdStandby` we use `Stop`? We do not really need a semantic distinction between `ColdStandby` and `Stop` in the Microgrid API.

Reply:
If we really don't need the distinction, I'm all for simplifying APIs.

So `cold standby == stop` and `hot standby == standby`?

Another option could be to have a boolean argument for standby that says if it's hot or cold.

Reply:
> So cold standby == stop and hot standby == standby?

`coldStandby` gets removed. We don't need it here. Going to `stop` is better.
Indeed  `hotStandby == standby`

>Another option could be to have a boolean argument for standby that says if it's hot or cold.

In general, actions that move components from one state to another should have dedicated methods. Using parameters to do it leads to mistakes. However, in this case, we just remove `coldStandby`.